AirCasting/Dashboard/DashboardView.swift

```diff
@@ -14,15 +14,15 @@ import Resolver
 struct DashboardView: View {
     @StateObject var coreDataHook: CoreDataHook
     @FetchRequest<SensorThreshold>(sortDescriptors: [.init(key: "sensorName", ascending: true)]) var thresholds
-    @EnvironmentObject var selectedSection: SelectSection
+    // TODO: - We can rethink the way how "Select Section" works and change it
+    @EnvironmentObject var selectedSection_: SelectSection
```

Comment:
I would rather name the new selectedSection a "selectedSectionForSwipingImplementation" or something like that because it is a confusing why there are two